src/TWIG/Blocks/card.twig

```diff
@@ -0,0 +1,12 @@
+<div class='card__content'>
+	<h4 class='title'>{{ title }}</h4>
+	<img class='image' src="{{ path }}" alt="{{ author }}">
+	<p class='description'>{{ description }}</p>
+	<div class='card__footer'>
+		<div class='author'>{{ author }}</div>
+		<div class='type'>{{ type }}</div>
+		<div class="link">
+			<a href="{{ link }}">{{ link }}</a>
+		</div>
+	</div>
+</div>
```

Comment:
Comment about all your files: Please adjust your IDE settings, you seem to have a tab symbol for all tabs instead of spaces, and your tab size is 3 or sometimes 4, please make sure your tab size is always = 2 and that you don't print tab symbol in all files